src/controller/like-controller.js

```diff
@@ -0,0 +1,27 @@
+import LikeService from '../service/like-service.js';
+
+class LikeController {
+  async addLike(req, res, next) {
+    try {
+      const groupId = parseInt(req.params.groupId, 10);
```

Comment:
```suggestion
      const { groupId } = req;
```
미들웨어에서 파싱한 값을 받아오면 이런 식으로 사용하시면 될 것 같습니다!